maildump/db.py

```diff
@@ -60,6 +60,15 @@ def create_tables():
     """)
 
 
+def message_get_parts(message):
```

Comment:
I think `iter_message_parts` would be a better name (`message_..._parts` sounds strange, and I usually like `iter` instead of `get` for generator functions)